src/DSi.cpp

```diff
@@ -312,6 +336,21 @@ bool LoadBIOS()
 
 bool LoadNAND()
 {
+    // Skip loading the stage2 if we started directly a ROM
+//    if (NDS::IsDirectBoot)
```

Comment:
doesn't this also belong into SetupDirectBoot?

Reply:
I would agree if we move the LoadNAND out of the Reset() and SofReset() as it messes up the NWRAM content and mapping, even tho this is not a hardware function of the reset.
Rather it is just another nds image just taken from a different location.

Maybe we should introduce a "SetupBottloader" parallel to the SetupDirectBoot?